utils.go

```diff
@@ -62,6 +62,58 @@ func DecodeLabel(image imgutil.Image, label string, v interface{}) error {
 	return nil
 }
 
+// RemoveStagePrefixes creates a slice with the build: or run: style prefixes removed.
+func RemoveStagePrefixes(mixins []string) []string {
```

Comment:
I don't think `lifecycle.RemoveStagePrefixes` should be exported from the lifecycle package.

If `pack` or another platform wants to consume this functionality from lifecycle as a lifecycle I think we should do either of the following instead
1. export a higher level method instead along the lines of `ValidateRebaseMixins`.
2. move utitlities into a mixins package so that the interface is more like`mixins.RemoveStagePrefixes`. In this case I would want to take the time to design what we think is the best library interface for mixins rather than taking this wholesale.

I inclined to say we can just leave this unexported for now.